build/FrameworkPackage/MakeFrameworkPackage.ps1

```diff
@@ -61,9 +61,25 @@ Copy-IntoNewDirectory "$inputBasePath\sdk\$inputBaseFileName.winmd" $fullOutputP
 Write-Verbose "Copying $inputBasePath\Themes"
 Copy-IntoNewDirectory -IfExists $inputBasePath\Themes $fullOutputPath\PackageContents\Microsoft.UI.Xaml
 
-[xml]$sdkPropsContent = Get-Content $PSScriptRoot\..\..\sdkversion.props
-$highestSdkVersion = ($sdkPropsContent.SelectNodes("//*[contains(local-name(), 'SDKVersion')]") | Where-Object { -not $_.ToString().Contains("Insider")} | Select-Object -Last 1).'#text'
-$sdkReferencesPath=$kitsRoot10 + "References\" + $highestSdkVersion;    
+#Find the latest available sdk
+function Get-SDK-References-Path
+{
+    [xml]$sdkPropsContent = Get-Content $PSScriptRoot\..\..\sdkversion.props
+    $sdkVersions = $sdkPropsContent.SelectNodes("//*[contains(local-name(), 'SDKVersion')]")
+    for ($i=$sdkVersions.Count-1; $i -ge 0; $i--)
```

Comment:
This depends on the fact that the sdk versions in sdkversion.props are sorted.

Add a comment to the .props file stating that they must remain sorted in ascending order.

Reply:
Updated the script to sort the version numbers in descending order.

Reply:
Even better.